boards/main/src/main.rs

```diff
@@ -157,12 +156,12 @@ mod app {
         )
     }
 
-    #[idle]
-    fn idle(_: idle::Context) -> ! {
-        loop {
-            rtic::export::wfi();
-        }
-    }
+    // #[idle]
```

Comment:
Probably shouldn't totally remove this idle function entirely, but WFI is an issue as this appears to cause problems with CAN and introduces Schrödinger's cat :)